projects/github.com/bcicen/ctop/package.yml

```diff
@@ -0,0 +1,30 @@
+distributable:
+  url: https://github.com/bcicen/ctop/archive/refs/tags/v{{version}}.tar.gz
+  strip-components: 1
+
+display-name: ctop
+
+versions:
+  github: bcicen/ctop
+
+build:
+  dependencies:
+    go.dev: ^1.18
+    git-scm.org: "*"
+
+  script: |
```

Comment:
i usually always use script arrays these days, since they're easier to alter and tweak, but this is just a suggestion

Reply:
can comment on script arrays!